web/src/message_edit.ts

```diff
@@ -988,11 +995,7 @@ export function start_inline_topic_edit($recipient_row: JQuery): void {
     const topic = message.topic;
     const $inline_topic_edit_input = $form.find<HTMLInputElement>("input.inline_topic_edit");
     $inline_topic_edit_input.val(topic).trigger("select").trigger("focus");
-    const $stream_topic = $recipient_row.find(".stream_topic");
-    const topic_width = $stream_topic.width();
-    // Set the width of the inline topic edit input to the
-    // width of the topic name + 1ch width for some cushion.
-    $inline_topic_edit_input.css("max-width", `calc(${topic_width}px + 1ch)`);
```

Comment:
It looks like this removes the `1ch` cushion; I'm not sure that's desirable?

Reply:
We have the cushion of `1ch` in `update_inline_topic_edit_input_max_width`, right ?

Reply:
Oh, I'd missed that appeared in two places. OK.